src/IQFeed.CSharpApiClient.Tests.Integration/Streaming/Level1/Level1ClientTests.cs

```diff
@@ -48,6 +49,42 @@ public async Task Should_Return_UpdateSummaryMessage()
             // Assert
             Assert.AreEqual(updateSummaryMessage.Symbol, Symbol);
         }
+        
+        [Test]
+        public async Task Should_Return_UpdateSummaryMessageWithDynamicFields()
+        {
+            // Arrange
+            var fieldNames = new DynamicFieldset[]
+            {
+                DynamicFieldset.Symbol, DynamicFieldset.MostRecentTrade, DynamicFieldset.MostRecentTradeSize, 
+                DynamicFieldset.MostRecentTradeTime, DynamicFieldset.MostRecentTradeMarketCenter,
+                DynamicFieldset.TotalVolume, DynamicFieldset.Bid, DynamicFieldset.BidSize, 
+                DynamicFieldset.Ask, DynamicFieldset.AskSize, DynamicFieldset.Open, DynamicFieldset.High, 
+                DynamicFieldset.Low, DynamicFieldset.Close, DynamicFieldset.MessageContents, 
+                DynamicFieldset.MostRecentTradeConditions, DynamicFieldset.MostRecentTradeAggressor,
+                DynamicFieldset.MostRecentTradeDayCode
+            };
+
+            var level1MessageHandler = new Level1MessageDynamicHandler();
+            var level1Client = Level1ClientFactory.CreateNew(level1MessageHandler);
+
+            try
```

Comment:
Simply because I was using Level1MessageDynamicHandler rather than the standard handler from the setup, so I created my own connection. It's a bit confusing though, I admit, so I've split that test out into a separate file now. Level1ClientDynamicFieldsetTests using the same format as the previous test file.